tests/unit/api/bucketPutVersioning.js

```diff
@@ -29,9 +31,26 @@ const locConstraintNonVersioned =
 '<LocationConstraint>withoutversioning</LocationConstraint>' +
 '</CreateBucketConfiguration>';
 
+const xmlReplicationConfiguration =
+'<ReplicationConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">' +
+    '<Role>arn:aws:iam::account-id:role/src-resource</Role>' +
+    '<Rule>' +
+        '<Prefix></Prefix>' +
+        '<Status>Enabled</Status>' +
+        '<Destination>' +
+            '<Bucket>arn:aws:s3:::destination-bucket</Bucket>' +
+            '<StorageClass>us-east-2</StorageClass>' +
+        '</Destination>' +
+    '</Rule>' +
+'</ReplicationConfiguration>';
+
 const externalVersioningErrorMessage = 'We do not currently support putting ' +
 'a versioned object to a location-constraint of type Azure or GCP.';
 
+const replicationVersioningErrorMessage = 'A replication configuration is ' +
```

Comment:
It's ok to test an error message but I'm not a big fan as it makes it more clumsy to change the wording (have to change the test as well), I usually prefer to just check the error code or type.

Reply:
Ok, sounds good to me. I was following the pattern of this test file, but I think that's a good enough reason to change it up.